docs/integrating-with-unity/defining-states.mdx

```diff
@@ -0,0 +1,334 @@
+---
+sidebar_label: 'Defining States'
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Defining States
+
+:::info
+TODO: We need to write a better description about why it is important to limit the state exposed to your bot.
+If we can't justify this vs accessing game object state directly, we should question what value we add...
+:::
+
+The state of the game sent to Regression Games is limited to only GameObjects selected to be included as part of the state.
+
+**A GameObject can be included as part of the state using one or more of the following methods.**
```

Comment:
I think this line would work better as a header instead of bold, as it doesn't look like its starting a list. Something like 'Ways to include state', with each of the methods being a sub-heading.

Reply:
updated